webui/react/src/pages/ExperimentDetails/CodeViewer/CodeViewer.tsx

```diff
@@ -341,35 +360,48 @@ const CodeViewer: React.FC<Props> = ({
                   * <Button className={css.noBorderButton}>Open in Notebook</Button>
                   */
                   <Tooltip title="Download File">
-                    {
-                      !String(activeFile.key).includes('Configuration') && (
-                        // hiding the download for configs until next iteration
-                        <DownloadOutlined
-                          className={css.noBorderButton}
-                          onClick={(e) => {
-                            const filePath = String(activeFile.key);
-                            if (filePath.includes('Configuration')) {
-                              const url = filePath.includes('runtime')
-                                ? URL.createObjectURL(new Blob([ runtimeConfig ]))
-                                : URL.createObjectURL(new Blob([ submittedConfig as string ]));
-
-                              handlePath(e, {
-                                external: true,
-                                path: url,
-                              });
-                            } else {
-                              handlePath(e, {
-                                external: true,
-                                path: paths.experimentFileFromTree(
-                                  experimentId,
-                                  String(activeFile.key),
-                                ),
-                              });
-                            }
-                          }}
-                        />
-                      )
-                    }
+                    <DownloadOutlined
+                      className={css.noBorderButton}
+                      onClick={(e) => {
```

Comment:
this re-declares the function every rendering, so we should `useCallback`.
Also, I dont think we should write logic in `onClick` directly, especially when it's long. Could you make a function for it?